indexing.go

```diff
@@ -123,7 +129,7 @@ func BulkIndex(docs []string, options Options) error {
 		// use it in the header.
 		if options.IDField != "" {
 			var docmap map[string]interface{}
-			dec := json.NewDecoder(strings.NewReader(doc))
+			dec := jsoniter.NewDecoder(strings.NewReader(doc))
```

Comment:
I tried jsoniter for another projects and switched back to encoding/json, since jsoniter could not handle cases encoding/json could handle just fine. Unfortunately, I haven't filed a bug report against jsoniter, I should do this some times, since it's a good project otherwise. The json encoding might also not be the top bottleneck at the moment.

So please revert jsoniter to encoding/json for the moment.